src/Matej.php

```diff
@@ -55,7 +55,7 @@ public function setHttpResponseDecoder(ResponseDecoderInterface $responseDecoder
         return $this;
     }
 
-    protected function getRequestManager(): RequestManager
+    public function getRequestManager(): RequestManager
```

Comment:
I'd prefer to keep this as protected. The request manager is internal implementation detail, you should not know about it at all, when you interact with `Matej` object - this will also break the encapsulation.

Maybe we can add  `setBaseUrl()` method to `Matej` object instead?